src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/AtomPub10CategoriesDocumentFormatter.cs

```diff
@@ -6,13 +6,15 @@
 namespace System.ServiceModel.Syndication
 {
     using System;
+    using System.Diagnostics.CodeAnalysis;
```

Comment:
What's pulling in the CodeAnalysis namespace?

Reply:
The code ported from desktop had `[SuppressMessage]` on some methods. I removed those attributes, but forgot to remove this using.